robottelo/cli/template.py

```diff
@@ -65,13 +60,20 @@ def add_operatingsystem(cls, options=None):
 
     @classmethod
     def remove_operatingsystem(cls, options=None):
-        """
-        Remove operating system, requires "id" and "operatingsystem-id".
-        """
-
+        """Remove operating system, requires "id" and "operatingsystem-id"."""
         cls.command_sub = 'remove-operatingsystem'
 
         result = cls.execute(
             cls._construct_command(options), output_format='csv')
 
         return result
+
+    @classmethod
+    def clone(cls, options=None):
+        """Clone provided provisioning template"""
+        cls.command_sub = 'clone'
+
```

Comment:
I would drop this blank line and return the line below directly instead of assigning to result and returning the result.